corpus/config_db.json

```diff
@@ -1,10 +1,63 @@
 [
+  {
+    "fields": {
+      "about": "The NITK-IEEE Computer Society is a conglomeration of students with a passion for Computer Science and Information Technology. This SIG is a great platform for those who want to gain expertise in computer based technology. \r\nIn the true spirit of divide and conquer, our community intends to explore and contribute to the rapidly evolving world of computer technologies by carrying out research and projects in the broad domains of computer science, in the form of 3 research groups : Intelligence, Systems and Core Computer Science.",
```

Comment:
This is the new about for Compsoc : 

"The NITK-IEEE CompSoc Special Interest Group is a conglomeration of Computer Society and Computational Intelligence Society (CIS). The SIG consists of students with a passion for Computer Science and Information Technology. This SIG is a great platform for those who want to gain expertise in computer based technology. \r\nIn the true spirit of divide and conquer, our community intends to explore and contribute to the rapidly evolving world of computer technologies by carrying out research and projects in the broad domains of computer science, in the form of 3 research groups : Intelligence, Systems and Core Computer Science."


Display 2 images (similar to DIode) for the hero section : CompSoc logo and CIS Logo.

CIS Logo can be found [here](https://cis.ieee.org/about/volunteer-resources/ieee-cis-logos)

Reply:
done, change made in db, and both light and dark images for both CIS and CompSoc are there in from b4 in the repo